MathlibTest/LintStyle.lean

```diff
@@ -119,6 +119,39 @@ lemma foo' : True := trivial
 
 -- TODO: add terms for the term form
 
+/--
+warning: Unscoped option maxHeartbeats is not allowed:
+Please scope this to individual declarations, as in
+set_option maxHeartbeats in
+-- comment explaining why this is necessary
+...
+note: this linter can be disabled with `set_option linter.style.setOption false`
```

Comment:
I parsed this incorrectly on the first reading. It needs to be:
```suggestion
warning: Unscoped option maxHeartbeats is not allowed:
Please scope this to individual declarations, as in
```
set_option maxHeartbeats in
-- comment explaining why this is necessary
...
```
note: this linter can be disabled with `set_option linter.style.setOption false`
```
at least, and even better would replace `...` with `example : ... := ...`

Reply:
Thanks - added the example.